src/Aspose.Imaging.Cloud.Sdk.Test/Api/FramesApiTests.cs

```diff
@@ -0,0 +1,471 @@
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright company="Aspose" file="FramesApiTests.cs">
+//   Copyright (c) 2018-2019 Aspose Pty Ltd. All rights reserved.
```

Comment:
Please, change the copyright period to 2020 in all the new classes

Reply:
I've updated the copyright dates.